mypy/main.py

```diff
@@ -179,6 +179,7 @@ def process_options(args: List[str],
 
     strict_flag_names = []  # type: List[str]
     strict_flag_assignments = []  # type: List[Tuple[str, bool]]
+    valid_disallow_any_options = {'unimported'}
```

Comment:
Yep, all that makes sense!

Yeah, I didn't think about the fact that the help output could be in different order -- that's a good point!